tests/parametric/test_otel_span_methods.py

```diff
@@ -71,7 +71,7 @@ def test_otel_set_service_name(self, test_agent, test_library):
         context.library == "java",
         reason="Old array encoding was removed in 1.22.0 and new span naming introduced in 1.24.0: no version elligible for this test.",
     )
-    @irrelevant(context.library < "golang@v1.59.0", reason="Old array encoding no longer supported")
+    @irrelevant(context.library >= "golang@v1.59.0", reason="New span naming introduced in v1.59.0")
```

Comment:
The CI is failing. We test `1.59.0.dev0` I guess the change is already present. To fix that : 

```suggestion
    @irrelevant(context.library >= "golang@v1.59.0.dev0", reason="New span naming introduced in v1.59.0")
```